ui/src/index.tsx

```diff
@@ -443,9 +449,7 @@ class Root extends PureComponent {
                             path="checks/:checkID"
                             component={CheckHistory}
                           />
-                          <Route path="about" component={OrgProfilePage}>
-                            <Route path="rename" component={RenameOrgOverlay} />
-                          </Route>
+                          <Route path="about" component={OrgProfilePage} />
```

Comment:
I'm confused about why we're setting the path here if it already exists here:

https://github.com/influxdata/influxdb/pull/17724/files#diff-58cea1fdba4d8d7058e35cb66db3a171R407

Reply:
The hierarchy was changed when the tree navigation was put in place.

**/orgs/:orgId/about** is the page that handles showing the Organization Profile, including the button that will rename the org.

**/orgs/:orgId/settings/about/rename** is where the overlay appears when renaming the org.

**/orgs/:orgId/settings/about** is the landing page after renaming an org.

All three are needed.